python/py-isce2/Portfile

```diff
@@ -6,21 +6,22 @@ PortGroup           python 1.0
 PortGroup           active_variants 1.1
 PortGroup           compilers 1.0
 
-github.setup        isce-framework isce2 2.3.1 v
+github.setup        isce-framework isce2 2.3.2 v
 revision            0
 name                py-isce2
 platforms           darwin
 license             Apache
 
 maintainers         {gps.caltech.edu:piyush @piyushrpt} \
+                    {univ-lyon1.fr:matthieu.volat @mazhe} \
```

Comment:
Why are you adding another co-maintainer here? Has @mazhe agreed to be a maintainer?

Reply:
Hi, yes, we did discuss this OOB, I've been following the discussion and I agree to be co-maintainer.

Reply:
OK great. 

@piyushrpt: Please add this to the commit message.

Reply:
Done.